### PR TITLE
Add Lithuanian translation

### PR DESCRIPTION
**Pull request by DINKONDBD**
I am translated whole site to Lithuanian language

### CODE REVIEW

File: lt/index.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
+<!DOCTYPE html>
+<html lang="lt">
+<head>
+  <title>Ne klauskite, kad klausti, klauskite iš karto</title>
+  <meta property="og:title" content="Ne klauskite, kad klausti, klauskite iš karto" />
+  <meta property="og:type" content="website" />
+  <meta property="og:url" content="https://dontasktoask.com/ru/" />
```
Comment (tulir) [inline lt/index.html:7]:
```suggestion
  <meta property="og:url" content="https://dontasktoask.com/lt/" />

```

File: lt/index.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    <p>
+      Susiję klausimai: <a href="https://xyproblem.info/">XY problema</a>, <a href="https://nohello.net/">Ne labas</a>.
+      Toliau skaitykite: <a href="https://stackoverflow.com/help/how-to-ask">kaip klausti gerą klausimą</a>,
+      arba jeigu jus turite daug laiko: <a href="http://catb.org/~esr/faqs/smart-questions.html">Kaip klausti klausimą protingas būdas</a>.
+    </p>
+  </main>
+  <footer>
+    -
+    
+    -
```
Comment (tulir) [inline lt/index.html:85]: These look unnecessary

```suggestion
```

File: lt/index.html [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    Kontentas daugiausiai paimtas iš <a href="https://iki.fi/sol/dontask.html">iki.fi/sol/dontask.html</a>
+	
+    -<a href="https://dontasktoask.com/">Originalas angliškai</a>
+	
+    <a href="https://github.com/maunium/dontasktoask.com">Kodas Github</a>
+  </footer>
```
Comment (tulir) [inline lt/index.html:91]: You can add translation credits to the footer and/or readme if you want

File: lt/index.html [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+		kad ir kuo tai baigtųsi,
+		net jei tai yra tikrai nėra labai susijęs su Java ir bet kas,
+		neturintis žinių apie Java ar galėtumėte atsakyti į mano klausimą?
+      </p>
+    </blockquote>
+    <p>
+		Yra daug priežasčių,kodėl žmonės, turintys žinių tinkamoje srityje,
+		gali neatsakyti.Tokiu būdu užduodami klausimą,jūs klausiate apie daugiau,nei
+		nei atrodo iš pirmo žvilgsnio.
+    </p>
```
Comment (TruncatedDinoSour) [inline lt/index.html:44]: take punctuation into consideration

and this shifts the meaning of the sentence a bit

replace 'tinkamoje' w 'tam tikroje, as 'tinkamoje' sounds a bit weird here

'klausiate apie daugiau' shifts the meaning of the sentence, the sentence should b like 'prašote daug daugiau nei manote' or smt like tht

File: lt/index.html [hunk 5]
```diff
@@ -0,0 +1,90 @@
...
+    <p>
+	Jūs klausiate žmonių, ar jie nori prisiimti atsakomybę. Tu abejoja jų pasitikėjimu savo įgūdžiais.
+	Taip pat žr.: nepagrįstai atsisakyti kitų žmonių pagalbos. Pavyzdžiui, aš dažnai atsakyti į klausimus,
+	susijusius su kalbomis ar bibliotekomis, kurias aš niekada nenaudojamas,
+	nes atsakymai yra tik sveikas protas (programuotojo požiūriu).
```
Comment (TruncatedDinoSour) [inline lt/index.html:49]: another place where ur mixing 'klausiate' and 'prašote', which shifts the meaning of the sentence

plus ur using incorrect pronouns and gendering ( e.g. Tu abejoja => Tu abejoji )

this sentence should b re-translated, idk if theres a proper word for libraries but my old informatics teacher used to call it 'biblioteka' so idk, feel free to find a better word :

> Jūs prašote žmonių prisiimti atsakomybę. Jūs keliate abejonių dėl žmonių gebėjimų. Jūs taip pat nereikalingai atskiriate kitus žmones. Dažnai atsakau į klausimus, susijusius su programavimo kalbomis ar bibliotekomis, kurių niekada nenaudojau, nes atsakymai (programuotojo požiūriu) yra visiems žinomi.

kinda a botched translation bc its hard to translate this properly

File: lt/index.html [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+    </p>
+    <p>	
+	Alternatyviai šį klausimą galima perskaityti kaip..
+    </p>
+    <blockquote>
+      <span class="name">Foobar123:</span>
+      <p class="message">
+		Turiu klausimą, susijusį su "Java", bet labai tingiu, kad jį suformuluočiau nebent čia yra kas nors,
+		kas galėtų į tai atsakyti.
```
Comment (TruncatedDinoSour) [inline lt/index.html:58]: sounds a bit weird w 'tai', try using 'jį' instead

smt like 'nebent čia yra kažkas, kas galėtų jį atsakyti'

File: lt/index.html [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+      </p>
+    </blockquote>
+    <p>
+		.. kuris tiesiog kalba apie jūsų tingumą. Jei nenorite atlikti darbo, Kodėl, norėdami išspręsti jūsų problemą,
+		turėtume to norėti?
```
Comment (TruncatedDinoSour) [inline lt/index.html:63]: this sentence also looks very weird and odd, i cant even make sense out of it rlly lol

instead, u should say smt like : 

> ... o tai tiesiog tingu. Jei nenorite atlikti darbo, kad išspręstumėte savo problemą, kodėl turėtume tai padaryti mes?

File: lt/index.html [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+    </p>
+    <p>
+		Sprendimas yra ne prašyti paklausti, o tiesiog paklausti. Kažkas kas palaiko pokalbį fone ir kartais žvilgčioja ten,
+		kad pamatytų, ką ten eina, vargu ar atsakys į jūsų klausimą "klausk paklausti",
+		tačiau jūsų tikrosios problemos aprašymas gali sukelti jų susidomėjimą ir priversti juos atsakyti.
```
Comment (TruncatedDinoSour) [inline lt/index.html:68]: should b re-translated as this makes no sense, heres the original sentence : 

> The solution is not to ask to ask, but just to ask. Someone who is idling on the channel and only every now and then glances what's going on is unlikely to answer to your "asking to ask" question, but your actual problem description may pique their interest and get them to answer.

File: lt/index.html [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+    </p>
+    <p>
+      Bendrai - ne klauskite
+      <em>"Kažkas iš čia raso Java kalba?"</em>,
+      Vietoj to paklauskite:
+      <em>"kaip padaryti [problema] naudojant Java kalba, ir [kita atitinkančia informacija]?"</em>
```
Comment (TruncatedDinoSour) [inline lt/index.html:74]: maybe instead of 'padaryti' this should b like 'atlikti' or 'išspręsti'

File: lt/index.html [hunk 10]
```diff
@@ -0,0 +1,90 @@
...
+    </p>
+    <p>
+      Susiję klausimai: <a href="https://xyproblem.info/">XY problema</a>, <a href="https://nohello.net/">Ne labas</a>.
```
Comment (TruncatedDinoSour) [inline lt/index.html:77]: 'ne labas' lmao

File: lt/index.html [hunk 11]
```diff
@@ -0,0 +1,90 @@
...
+      Toliau skaitykite: <a href="https://stackoverflow.com/help/how-to-ask">kaip klausti gerą klausimą</a>,
+      arba jeigu jus turite daug laiko: <a href="http://catb.org/~esr/faqs/smart-questions.html">Kaip klausti klausimą protingas būdas</a>.
```
Comment (TruncatedDinoSour) [inline lt/index.html:79]: 'Kaip protingau klausti klausimą' would b correct, this just looks weird xD

File: lt/index.html [hunk 12]
```diff
@@ -0,0 +1,90 @@
...
+    </p>
+  </main>
+  <footer>
+    Kontentas daugiausiai paimtas iš <a href="https://iki.fi/sol/dontask.html">iki.fi/sol/dontask.html</a>
```
Comment (TruncatedDinoSour) [inline lt/index.html:83]: turinys, not kontentas

**Review by TruncatedDinoSour — CHANGES_REQUESTED** [review]
please fix the translation :)

**Comment by TruncatedDinoSour** [comment]
sad

**Pull request closed without merging**